snap/snapcraft.yaml

```diff
@@ -121,7 +121,21 @@ plugs:
       - /etc/pulse
       - /etc/shells
       - /etc/xdg/autostart
+      - /etc/xdg/user-dirs.conf
+      - /etc/xdg/user-dirs.defaults
```

Comment:
We'll need to get the snap declaration updated to get this through. It might be lower effort to add this access to as permanent slot rules for the desktop interface.

Reply:
> We'll need to get the snap declaration updated to get this through. It might be lower effort to add this access to as permanent slot rules for the desktop interface.

@jhenstridge would this be sufficient?  https://github.com/canonical/ubuntu-core-desktop-snapd/pull/12